docs/monitoring/profiling.md

```diff
@@ -113,3 +113,20 @@ For example:
 $ curl http://localhost:8080/debug/pprof/heap > /tmp/heap-seed-admission-controller
 $ go tool pprof /tmp/heap-seed-admission-controller
 ```
+
+## gardener-resource-manager
```

Comment:
As enabling profiling for GSAC and GRM is exactly the same, we could combine both components into 1 sections such as:

```diff
--- a/docs/monitoring/profiling.md
+++ b/docs/monitoring/profiling.md
@@ -96,9 +96,9 @@ $ curl http://localhost:2723/debug/pprof/heap > /tmp/heap-admission-controller

-## gardener-seed-admission-controller
+## gardener-seed-admission-controller, gardener-resource-manager

-gardener-seed-admission-controller provides the following flags for enabling profiling handlers (disabled by default):
+gardener-seed-admission-controller and gardener-resource-manager provide the following flags for enabling profiling handlers (disabled by default):

@@ -107,26 +107,9 @@ gardener-seed-admission-controller provides the following flags for enabling pro

 The handlers are served on the same port as configured in the `--metrics-bind-address` flag (defaults to `":8080"`) via HTTP.

-For example:
+For example (gardener-seed-admission-controller):
```

You can also notice that in this doc components with similar enablement are combine (gardener-admission-controller + gardener-scheduler and gardener-controller-manager + gardenlet).